docs/building-spark.md

```diff
@@ -276,34 +276,6 @@ Enable the profile (e.g. 2.13):
     # For sbt
     ./build/sbt -Pscala-2.13 compile
 
-## Running Jenkins tests with GitHub Enterprise
```

Comment:
Hmmm .. yeah Apache Spark itself use this script for now as far as I know but wouldn't it be worthwhile keeping it in case other people build their own Jenkins? To be clear, I am fine with removing this out but I wouldn't remove if anyone actually has a concern or uses it.

Reply:
As the script is still there, I'm also not sure why we need to remove this doc? Maybe it is better to update it as Apache Spark doesn't use it to report Jenkins tests to Github anymore, but the script is used to do that and still available to users?